README-dev.md

```diff
@@ -66,7 +66,9 @@ direnv with the `.envrc` provided. This devshell provides all the dependencies r
 
 ## Building Bindings
 
-To regenerate the OCaml and WebAssembly artifacts, you can do so within the o1js repo. The [bindings](https://github.com/o1-labs/o1js-bindings) and [Mina](https://github.com/MinaProtocol/mina) repos are both submodules of o1js so you can build them from within the o1js repo.
+To regenerate the OCaml and WebAssembly artifacts, you can do so within the
```

Comment:
I think we can RM the entire section "Building Bindings".  The sections for external and internal developers covers what must be known.